README.md

```diff
@@ -47,7 +47,7 @@ a section below showing how to properly compile it. The following command mirror
 cluster a Container Image hosted in quay.io:
 
 ```
-$ kubectl image import operator --from quay.io/shipwright/imgctrl:latest --mirror
+$ kubectl image import operator --source quay.io/shipwright/imgctrl:latest --mirror
```

Comment:
(not blocking) If `--source` is a required flag, can it just be a positional arg?

```
kubectl image import operator quay.io/shipwright/imgctrl:latest
```

Reply:
It is not a required flag at this stage. If not set then whatever Source is defined in the Image object is used. It is required if the Image object does not exist yet though.